frontend/src/dataUtilities.ts

```diff
@@ -12,12 +12,15 @@ import {
   AssetReport,
 } from "plaid/dist/api";
 
-const formatter = new Intl.NumberFormat("en-US", {
-  //should look like "$50" or "$50.34"
-  style: "currency",
-  currency: "USD",
-  minimumFractionDigits: 0,
-});
+const formatCurrency = (
+  number: number | null | undefined,
+  code: string | null | undefined
+) => {
+  if (number != null && number !== undefined) {
+    return ` ${parseFloat(number.toFixed(2)).toLocaleString("en")} ${code}`;
```

Comment:
spacing should probably be taken care of in the dom where this value is displayed instead of adding a leading space in the string. Also should the currency code come first? e.g. `£50.00`

Reply:
it's not actually a Euro or dollar symbol, but letters, so it's 50 Euro, so I think it should come afterwards?